package.json

```diff
@@ -9,7 +10,7 @@
   "scripts": {
     "clean": "rm -rf ./dist ./coverage",
     "eslint": "eslint --ext=js,md ./ --cache",
-    "build": "babel src --out-dir dist --source-maps",
+    "build": "babel src --out-dir dist --source-maps --presets es2015",
```

Comment:
@Ardeshir81 We use `babel-preset-env` for `babel`. Please see https://github.com/Fitbit/angular-translate-loader/blob/master/.babelrc

Could you please remove `--presets` flag, because it looks unnecessary?